docs/sources/enterprise/license/license-restrictions.md

```diff
@@ -22,25 +22,29 @@ In the context of licensing, each user is classified as either a viewer or an ed
 
 - An editor is a user who has permission to edit and save a dashboard. Examples of editors are as follows:
     - Grafana server administrators.
-    - Users who are assigned an organizational role of Editor or Admin.
-    - Users that have been granted Admin or Edit permissions at the dashboard or folder level. Refer to [Dashboard and folder permissions]({{< relref "../../permissions/dashboard-folder-permissions.md" >}}).
+    - Users who are assigned an organization role of Editor or Admin.
+    - Users who have been granted admin or edit permissions at the dashboard or folder level. Refer to [Dashboard and folder permissions]({{< relref "../../permissions/dashboard-folder-permissions.md" >}}). This means that even if a user has the role "viewer," they will be counted as an editor if they have permission to edit/admin a dashboard or folder.
```

Comment:
```suggestion
    - Users who have been granted admin or edit permissions at the dashboard or folder level. Refer to [Dashboard and folder permissions]({{< relref "../../permissions/dashboard-folder-permissions.md" >}}). This means that even if a user is assigned to an organization role of Viewer they will be counted as an editor.
```
Two minor suggestion:
1. Say "if a user is assigned to an organization role of Viewer" instead of "if a user has the role "viewer," to be consistent within this doc.
2. Remove "if they have permission to edit/admin a dashboard or folder" as it is already mentioned in the first sentence.